### PR TITLE
Added test for ListsingsManager.DeleteListingsProvider().

### PR DESCRIPTION
**Pull request by kennethcochran**
**Changes**
Addition of ListingsManagerTests to JellyFin.LiveTv.Tests project.
Addition of test for ListsingsManager.DeleteListingsProvider() to ListingsManagerTests.

**Issues**
N/A

### CODE REVIEW

File: tests/Jellyfin.LiveTv.Tests/Listings/ListingsManagerTests.cs [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+using Jellyfin.LiveTv.Listings;
+using MediaBrowser.Common.Configuration;
+using MediaBrowser.Controller.LiveTv;
+using MediaBrowser.Model.LiveTv;
+using MediaBrowser.Model.Tasks;
+using Microsoft.Extensions.Logging;
+using Moq;
+using Xunit;
+
+namespace Jellyfin.LiveTv.Tests.Listings
```
Comment (Bond-009) [inline tests/Jellyfin.LiveTv.Tests/Listings/ListingsManagerTests.cs:11]: Use file scoped namespaces for new files

File: tests/Jellyfin.LiveTv.Tests/Listings/ListingsManagerTests.cs [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    {
+        private readonly IConfigurationManager _config;
+        private readonly IListingsProvider[] _listingsProviders;
+        private readonly ILogger<ListingsManager> _logger;
+        private readonly ITaskManager _taskManager;
+        private readonly ITunerHostManager _tunerHostManager;
+
+        public ListingsManagerTests()
+        {
+            // Mock dependencies
```
Comment (Bond-009) [inline tests/Jellyfin.LiveTv.Tests/Listings/ListingsManagerTests.cs:23]:
```suggestion
```

File: tests/Jellyfin.LiveTv.Tests/Listings/ListingsManagerTests.cs [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+            Mock.Get(_config)
+                .Setup(x => x.GetConfiguration(It.IsAny<string>()))
+                .Returns(new LiveTvOptions { ListingProviders = [new ListingsProviderInfo { Id = id }] });
+
+            // Act
+            manager.DeleteListingsProvider(id);
+
+            // Assert
+            Assert.DoesNotContain(_config.GetLiveTvConfiguration().ListingProviders, p => p.Id == id);
```
Comment (Bond-009) [inline tests/Jellyfin.LiveTv.Tests/Listings/ListingsManagerTests.cs:46]: use string.Equals when comparing strings

**Review by Bond-009 — CHANGES_REQUESTED** [review]

**Review by Bond-009 — APPROVED** [review]

**Review by JPVenson — APPROVED** [review]

**Review by Shadowghost — APPROVED** [review]
@kennethcochran can you please fix the merge conflict?

**Review by JPVenson — APPROVED** [review]

**Pull request merged**